.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
-  - lts/dubnium
```

Comment:
I was under the impression that dubnium does support `.mjs`? It didn’t work for you?

Might it be better to set another lower barrier? Erbium?

Reply:
If possible, I would prefer to still test on dubnium

Reply:
Erbium is passed.

dubnium works with `--experimental-modules` flag though .cjs extension is not supported when importing from .mjs.
Anyway this should not affect node 10 users if you stick with es5. We just should not add .cjs proxies.

Reply:
> Erbium is passed.

What do you mean with this?

Reply:
oh whoops, sorry, missed that erbium is below this line.

Reply:
Erbium tests are passed :)